dotcom-rendering/src/web/components/TopicFilterBank.importable.tsx

```diff
@@ -66,6 +66,11 @@ const handleKeyEventClick = (filterKeyEvents: boolean) => {
 	window.location.search = urlParams.toString();
 };
 
+const availableTopicIsSelected =
```

Comment:
I'm tempted to name this isEqual as it's just defining whether we consider two topics to be the same?

Reply:
great suggestion, do you also think the param names should change? Or `selectedTopic` & `availableTopic` are fine?

Reply:
+1 to isEqual but I think we can keep the param names as is.